.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.preferences.importModuleSpecifierEnding": "js"
```

Comment:
I just realized I set this per your recommendation but with `javascript.` 🤦 That's why it was not working.